mm2src/coins/lp_coins.rs

```diff
@@ -404,6 +415,30 @@ pub struct RawTransactionRes {
     pub tx_hex: BytesJson,
 }
 
+#[derive(Clone, Debug, Deserialize)]
+pub struct PrevTxns {
+    pub tx_hash: String,
+    pub index: u32,
+    pub script_pub_key: String,
+    pub redeem_script: Option<String>,
+    pub amount: BigDecimal,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct SignRawTransactionRequest {
+    pub coin: String,
+    pub tx_hex: String,
+    pub prev_txns: Option<Vec<PrevTxns>>,
```

Comment:
I don't think `prev_txns` should be made optional since it's a `vector` . I mean we can just have `Vec<PrevTxns>` and instead of checking for `Some(prev_txns) = prev_txns` we can just do `prev_txns.is_empty()`

Reply:
I wanted to have this rpc similar to the same one in the daemon where the same param is optional.
Having the prev_txns param as optional allows to ignore it completely meaning the rpc should rely on txns in the chain

Reply:
Ok make sense. thanks